src/storage/disk_table.h

```diff
@@ -0,0 +1,365 @@
+/*
+ * Copyright 2021 4Paradigm
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <atomic>
+#include <map>
+#include <memory>
+#include <string>
+#include <vector>
+#include "base/endianconv.h"
+#include "base/slice.h"
+#include "boost/lexical_cast.hpp"
+#include "common/timer.h"  // NOLINT
+#include "gflags/gflags.h"
+#include "proto/common.pb.h"
+#include "proto/tablet.pb.h"
+#include "rocksdb/compaction_filter.h"
+#include "rocksdb/db.h"
+#include "rocksdb/filter_policy.h"
+#include "rocksdb/options.h"
+#include "rocksdb/slice.h"
+#include "rocksdb/slice_transform.h"
+#include "rocksdb/status.h"
+#include "rocksdb/table.h"
+#include "rocksdb/utilities/checkpoint.h"
+#include "storage/iterator.h"
+#include "storage/table.h"
+
+namespace openmldb {
+namespace storage {
+
+static const uint32_t TS_LEN = sizeof(uint64_t);
+static const uint32_t TS_POS_LEN = sizeof(uint8_t);
```

Comment:
`ts_col->GetId()` return the seq number in schema. So if the number of schema fields great than 256, it may not run as expect. it's better to add the test case covering this scene.